src/Login.css

```diff
@@ -0,0 +1,63 @@
+
+
+.Login {
+    font-family: 'Montserrat', sans-serif;
```

Comment:
is there a way to mention the font family once throughout the entire file so you don't have to write it every time (for .Login h1, .Login input, etc)? you know, the DRY principle! Gotta make things dry 💯 I don't think we're going to use another font so :)

Reply:
Update: Pretty sure all HTML elements can carry a class attribute. You can prob create an overall class and then specify the font there because then all your elements have an overall class attribute (font-family: 'Montserrat', sans-serif) . Try this code: ```* {
font-family: 'Montserrat', sans-serif
}
```
the asterisk makes sure that all elements share the same attributes.